kubetest/util.go

```diff
@@ -88,8 +88,27 @@ func getLatestClusterUpTime(gcloudJSON string) (time.Time, error) {
 // getLatestGKEVersion will return newest validMasterVersions.
 // Pass in releasePrefix to get latest valid version of a specific release.
 // Empty releasePrefix means use latest across all available releases.
-func getLatestGKEVersion(project, zone, releasePrefix string) (string, error) {
-	res, err := control.Output(exec.Command("gcloud", "container", "get-server-config", fmt.Sprintf("--project=%v", project), fmt.Sprintf("--zone=%v", zone), "--format=value(validMasterVersions)"))
+func getLatestGKEVersion(project, zone, region, releasePrefix string) (string, error) {
+	cmd := []string{
+		"container",
+		"get-server-config",
+		fmt.Sprintf("--project=%v", project),
+		"--format=value(validMasterVersions)",
+	}
+
+	// --gkeCommandGroup is from gke.go
+	if *gkeCommandGroup == "alpha" || *gkeCommandGroup == "beta" {
```

Comment:
Can't we append it unconditionally. We used to use e.g. "beta --quiet" at some point :)

Reply:
changed to append whenever gkeCommandGroup is not empty